src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -118,6 +121,9 @@ export default memo(function LoadoutBuilder({
     },
     lbDispatch,
   ] = useLbState(stores, defs, preloadedLoadout, storeId);
+  // For compatibility with LoadoutEdit components
+  const setLoadout = (updateFn: LoadoutUpdateFunction) =>
+    lbDispatch({ type: 'setLoadout', updateFn });
```

Comment:
This currently allows smuggling Artifice mods into the mods list, which should not be allowed -- the worker process assumes it can freely assign all artifice mods.

Reply:
Good catch. I should really get some Artifice armor someday. I think it's fixed now.

Reply:
The randomize / sync from equipped buttons from the mods list still copy artifice mods.

Reply:
OK now I think I really got it. Artifice mods are stripped at the reducer, so they can never make it into the list.

Reply:
Not seeing a new commit - either GitHub is slow or you forgot to push?

Reply:
Yeah just forgot to push apparently